cds/modules/webhooks/receivers.py

```diff
@@ -292,6 +292,7 @@ def _init_object_version(self, event):
                 version_id = str(object_version.version_id)
             # and a tag associated to him
             ObjectVersionTag.create(object_version, '_event_id', event_id)
+            ObjectVersionTag.create(object_version, 'is_master', True)
```

Comment:
we should not add a new tag just because is convenient, otherwise we will have to add it to every single file we add to the record, i.e subtitles, as they are masters too.

We already know which files are master files, the ones which don't have a `master` key 😉